HLTrigger/JSONMonitoring/test/testHLTriggerJSONMonitoring_cfg.py

```diff
@@ -1,7 +1,9 @@
 import FWCore.ParameterSet.Config as cms
 
 process = cms.Process("TEST")
+
 process.options.numberOfThreads = 1
+process.options.numberOfStreams = 0
```

Comment:
Do you really want to define and then overwrite `numberOfThreads`?

Reply:
Mh.. this is number of streams?

Reply:
> Mh.. this is number of streams?

You are right: shame on me!